keyboards/tada68/keymaps/rys/config.h

```diff
@@ -0,0 +1 @@
+#include "../../config.h"
```

Comment:
This line is not actually needed, and should be removed. 

Specifically, the build script automatically includes all of the config.h files, so this is not necessary. 

But you should replace it with `#pragma once`, so the fine is included only once.